packages/gotrue/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.8.0
+
+ - **FIX**: reformat Provider type to be ordered alphabetically ([#471](https://github.com/supabase/supabase-flutter/issues/471)). ([c3a1dbb3](https://github.com/supabase/supabase-flutter/commit/c3a1dbb3974bcb7d95160d6412810c69122f2755))
```

Comment:
This should be removed here and in the global changelog, because that's not relevant.

Reply:
Let's not be too picky about the changelogs, especially the global one. The change did happen, and I think it should be left there. If anything, I should have noticed before merging the original PR that the order was not alphabetical.

Reply:
Okay